bin/evidence_string_generation.py

```diff
@@ -1,24 +1,16 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
Is this advisable given that the script has only been tested for compatibility since 3.5 and above?

Reply:
Well, I mean, `/usr/bin/python` is even worse because, depending on the system configuration, it will probably start Python 2 :-) At least `#!/usr/bin/env python3` is a bit more sensible shebang. And we invoke it directly as `python3.5 bin/evidence_string_generation.py` anyway.